src/containers/FilmFrontpage/MovieCategory.tsx

```diff
@@ -10,15 +10,17 @@ import React from 'react';
 import { CarouselAutosize } from '@ndla/carousel';
 //@ts-ignore
 import { CalculatedCarouselProps, FilmMovieList, MovieGrid } from '@ndla/ui';
+import { gql } from '@apollo/client';
 import { WithTranslation, withTranslation } from 'react-i18next';
 import { breakpoints, findName } from './filmHelper';
-import { GQLMovieTheme } from '../../graphqlTypes';
+import { GQLMovieCategoryThemeFragment } from '../../graphqlTypes';
```

Comment:
Så du må først eksportere en fragment i fila, så kjøre `yarn generate-gql-types` og så kan du importere den ferdiggenererte fragmenten?

Reply:
Nei, fragmenten trenger ikke å genereres. `yarn generate-gql-types` lager bare typescript-typer. Hvis man har behov for typen i komponenten fragmenten defineres i vil man derimot trenge å kjøre `generate-gql-types`. 

Et alternativt forslag er å lage en komponent-type som spesifiserer `fragments` som et statisk felt på en komponent. Da trenger man ikke mer enn én import for hver komponent. Da kan man istedenfor bare skrive `MovieCategory.fragments.theme`.